### DISCUSSION TITLE
Is there anyway that I could do the installation process manually as I am facing problems.

### DISCUSSION THREAD

**Discussion by abdelrahmanbonna** [lllyasviel/Fooocus#3188] [Q&A] [answered]
I tried to install different presets, but all of them failed because of a connection reset I need to have another way to install Fooocus on my machine.

install keeps freezing in here:
![image](https://github.com/lllyasviel/Fooocus/assets/2415646/3e266a77-a52b-4704-b1bf-c8fce60e9e35)

**Comment by Rogala** [comment, marked as answer]
Disable VPN or disable the firewall for a while, see how it will be.

**Reply by abdelrahmanbonna** [reply]
I tried to disable my firewall but it didn’t work. You could close this question as I did some trials and It worked for me when I removed the pause line in the bat file.

**Reply by mashb1t** [reply]
for persons debugging this in the future: you may also use the hf-mirror attribute and define to use a mirror website instead to optimize network traffic.
